front-end-side/src/Pages/Data.js

```diff
@@ -71,102 +71,112 @@ const EnterData = () => {
     return (
         <div>
             <Navbar />
+            {
+                newExpenses !== 0 ?
```

Comment:
We are getting _Upload your files or enter data to get statistics..._ for a second even if we have uploaded data. Could we fix it?